dvc/remote/local.py

```diff
@@ -56,7 +56,7 @@ def cache_dir(self, value):
     def supported(cls, config):
         return True
 
-    def list_cache_paths(self):
+    def list_cache_paths(self, prefix=None):
         assert self.path_info is not None
```

Comment:
```suggestion
        assert prefix is None
        assert self.path_info is not None
```
just to make sure that no one will try to use it.

Reply:
fixed in f17a0ad